tests/core/test_events_reason.py

```diff
@@ -0,0 +1,156 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
+import pytest
+
+from unittest.mock import ANY
+
+from parsec.core.core_events import CoreEvent, FSEntryUpdatedReason
+
+
+@pytest.mark.trio
+async def test_fs_update_event_reason(alice_user_fs, running_backend):
```

Comment:
super cool test 👍 🥇 😄

Reply:
Can you add a test for the file access with a file descriptor too ? (`fd_resize` and `fd_write`)

Reply:
I also add some test for workspace creation etc